azure-pipelines.yml

```diff
@@ -0,0 +1,49 @@
+jobs:
+- job:
+
+  strategy:
+    matrix:
+      Mojave-Xcode-10.2.1:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '10.2.1'
+      Mojave-Xcode-11.0:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '11'
+
+  pool:
+    vmImage: $(IMAGE_POOL)
+
+  steps:
+
+  - script: |
+      sudo xcode-select --switch "/Applications/Xcode_$(XCODE_VERSION).app/Contents/Developer"
+    displayName: "Select Xcode $(XCODE_VERSION)"
+
+  - script: xcodebuild -version
+    displayName: "Version of xcodebuild"
+  
+  - script: git clone -b develop https://github.com/calabash/FBSimulatorControl.git ../FBSimulatorControl
+    displayName: Download FBSimulatorControl dependency
+
+  - script: |
+      git clone -b master https://$(GitHubAccessToken):x-oauth-basic@github.com/xamarinhq/calabash-codesign.git ../calabash-codesign
```

Comment:
In multi-line scripts, always start with:

```
set -e
```

to force the script to exit if a subcommand failes.

Reply:
> ../calabash-codesign

I don't think we own that directory so we cannot put files there.

> $(GitHubAccessToken)

What GitHub Service Connection are you using?

Reply:
1. Added `set -e`. Thanks!
2. Replaced to `./calabash-codesign/`
3. `$(GitHubAccessToken)` - is a secret variable defined in build definition. It's store private access token of Github user who have pull-access to the repository.